### PR TITLE
Run CI builds on multiple OTP releases

### PR DESCRIPTION
**Pull request by bjosv**

### CODE REVIEW

File: test/ered_SUITE.erl [hunk 1]
```diff
@@ -489,7 +491,7 @@ t_ask_redirect(_) ->
     ?MSG(#{msg_type := slot_map_updated}, 1000),
 
     %% wait a bit to let the config spread to all nodes
-    timer:sleep(5000),
+    timer:sleep(10000),
```
Comment (zuiderkwast) [inline test/ered_SUITE.erl:494]: In Redis TCL tests, they use `wait_for_cluster_propagation` defined like this:

```TCL
proc wait_for_cluster_propagation {} {
    wait_for_condition 50 100 {
        [cluster_config_consistent] eq 1
    } else {
        fail "cluster config did not reach a consistent state"
    }
}

# (...)

proc cluster_config_consistent {} {
    for {set j 0} {$j < [llength $::servers]} {incr j} {
        if {$j == 0} {
            set base_cfg [R $j cluster slots]
        } else {
            if {[R $j cluster slots] != $base_cfg} {
                return 0
            }
        }
    }

    return 1
}
```

Comment (zuiderkwast) [reply test/ered_SUITE.erl:494]: I.e. wait until all nodes return the same result for `CLUSTER SLOTS`.

Comment (bjosv) [reply test/ered_SUITE.erl]: Good idea, I reverted the sleep change and fixed the issue in start_cluster/1 before I saw this comment.

Before running each testcase there was an existing check that each node-id hash was received in the slot-map.
Sometime the first `slot_map_message` might not contain the final slotmap period, and there is a need to get next message instead. Probably due to the periodic slot updated test each 500ms until the cluster is ok.

The problem is most often seen after `t_ask_redirect` where this change was done, but there were more testcase seemingly with the same problem..

Comment (bjosv) [reply test/ered_SUITE.erl]: Fixed in https://github.com/Ericsson/ered/pull/9

**Pull request merged**